apps/evm-profile/src/utils/IPFS.ts

```diff
@@ -0,0 +1,85 @@
+import { PinataPinResponse } from '@pinata/sdk';
+import { FileError, JsonError, ProfileMetadata, Result } from '../types';
+
+export const uploadJSON = async (
+  json: ProfileMetadata,
+): Promise<Result<PinataPinResponse, JsonError | string>> => {
+  try {
+    const response = await fetch('/api/json', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(json),
+    });
+
+    if (!response.ok) {
+      return { ok: false, error: response.statusText };
+    }
+
+    try {
+      const data = await response.json();
+      return { ok: true, value: data };
+    } catch (e) {
+      console.log(e);
+      return { ok: false, error: JsonError.JSONParseError };
+    }
+  } catch (e) {
+    console.log(e);
```

Comment:
Let's remove the console.log. The caller can choose to log the message out via the error handling if they want

If you were to console log it then you should use `console.error`, but no need to do this here